### DISCUSSION TITLE
Brace around parameter type annotations should be italics

### DISCUSSION THREAD

**Discussion by sscherfke** [pradyunsg/furo#571] [Potential Issue] [answered]
Currently, the braces around a callable's parameter's type annotations are straight, which looks ugly in combination with the italic type hints:

<img width="706" alt="grafik" src="https://user-images.githubusercontent.com/511179/200290881-375f4c05-069f-498e-b2c4-3064925ec73e.png">

I think it would typographically be more correct to make the braces italic, too.

**Comment by pradyunsg** [comment, marked as answer]
It looks ugly, but it's literally the markup that Sphinx generates.

Please consider filing an issue against Sphinx itself (after checking for duplicates) about this, to fix the markup generated by Sphinx.

Honestly, I'd prefer if it weren't italic at all and the whole thing were in line code like all other references to types, but Sphinx doesn't generate that. Further, the markup is such that it is not possible for me to style it like that.
